docs/launchd.md

```diff
@@ -0,0 +1,33 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Launchd background services on macOS.
+
+`tezos-accuser-<proto>`, `tezos-baker-<proto>`, `tezos-endorser-<proto>` formulas
+provide backround services for running the corresponding daemons.
+
+Since `tezos-node` and `tezos-signer` need multiple services they are provided
+in dedicated meta-formulas. These formulas don't install any binaries and only add
+background services.
+
+Formulas with `tezos-node` background services:
+* `tezos-node-mainnet`
+
+Formulas with `tezos-signer` background services:
+* `tezos-signer-http`
+* `tezos-signer-https`
+* `tesos-signer-tcp`
+* `tezos-signer-unix`
+
+To start the service: `brew services start <formula>`.
+
+To stop the service: `brew services stop <formula>`.
+
+All of the brew services have various configurable env variables. These variables
+can be changed in the corresponding `/usr/local/Cellar/tezos-signer-tcp/<version>/homebrew.mxcl.<formula>.plist`.
+Once the configuration is updated, you should restart the service:
+`brew services restart <formula>`.
+
+Note, that all services are run as a user agents, thus they're stopped after the logout.
```

Comment:
Perhaps this should be put in a bit more evidence, as it can be quite painful it if happens by surprise.